### DISCUSSION TITLE
"Accept sessions via permanent password" ignored when started as Service

### DISCUSSION THREAD

**Discussion by liquiddandruff** [rustdesk/rustdesk#6693] [General]
### Bug Description

The host is configured to use a permanent password.

When the RustDesk GUI is open and listening for connections, a peer that has previously connected, will connect automatically with the permanent password.

However when the RustDesk GUI is closed and the RustDesk Service is started instead, a peer will not be able to connect. The host reverts to "Use one-time password" and prompts the peer for the password.

It seems when started via the service, RustDesk is ignoring the configured permanent password setting.

### How to Reproduce

Set permanent password in GUI

Shutdown GUI

Start via Service

Attempt to connect from peer, receive password prompt.

### Expected Behavior

RustDesk Server should use same configuration settings as RustDesk GUI

### Operating system(s) on local side and remote side

Android -> Windows 11

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

N/A

### Additional Context

This is over RustDesk Server 1.1.9

**Comment by adamz01h** [comment]
On windows.
I am able to reproduce. The config file @ `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config\RustDesk.toml` that holds the password does not get updated by using
```ps
  .\rustdesk.exe --password $rustdesk_pw
```

**Reply by adamz01h** [reply]
I tested with my install script https://gist.github.com/adamz01h/f1f11b2ab8127b615af6791d8acb15bb
The parts for the service install are commented out.

**Reply by adamz01h** [reply]
updated Script

**Comment by adamz01h** [comment]
Corrected. The service and the program has to be running to set the password via CLI
